src/App.css

```diff
@@ -1,3 +1,24 @@
-.App {
-  text-align: center;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
- [ ] I would suggest you, please fix the spacing use the same color as in the original design. See below for reference.
### Your design
![his](https://github.com/hamzaw9/book-store/assets/81727988/99ab7ada-076e-42b9-82d6-78175a352533)

### Original design
![original](https://github.com/hamzaw9/book-store/assets/81727988/a7c4a050-84fe-4e5f-8918-fdb016e50ca7)